Meta/run.sh

```diff
@@ -212,7 +216,11 @@ fi
 
 if [ -z "$SERENITY_MACHINE" ]; then
     if [ "$SERENITY_ARCH" = "aarch64" ]; then
-        SERENITY_MACHINE="-M raspi3 -serial stdio"
+        if [ "$(uname)" = "Darwin" ]; then
```

Comment:
I'm not sure if `raspi3b` is Darwin specific but it is maybe possible to do this:
 ```
if $SERENITY_QEMU_BIN -M help | grep -q raspi3b; then
    SERENITY_MACHINE="-M raspi3b -serial stdio"
else
    SERENITY_MACHINE="-M raspi3 -serial stdio"
fi
```

Reply:
FWIW, I have the same issue on Linux.

A quick search shows that the `raspi3` machine type was just an alias for `raspi3b`, and has been removed in [QEMU 6.2](https://wiki.qemu.org/ChangeLog/6.2):
> The deprecated machine names 'raspi2' and 'raspi3' have been removed; use 'raspi2b' and 'raspi3b' instead

Reply:
great, I'll make this platform independent then